src/bde_copy.cpp

```diff
@@ -100,6 +100,7 @@ bool col_headers = false;
 bool nometa = false;
 bool use_archive = false;
 bool use_gzip = false;
+bool output = false;
```

Comment:
The name of this variable is too generic, what does it mean ?
Could you add a comment to document what it is for ?

Reply:
I have renamed the variable and added a comment to describe what the variable is for.